cdmi-stub/imp.h

```diff
@@ -88,8 +88,8 @@ class CMediaKeys : public IMediaKeys
     virtual ~CMediaKeys(void);
 
     virtual bool IsTypeSupported(
-        const char *f_pwszMimeType,
-        const char *f_pwszKeySystem) const;
+        const char *f_pwszKeySystem,
+        const char *f_pwszMimeType) const;
```

Comment:
Why swap the order of the parameters?

Reply:
To match the function call in cdmi/service.cpp, line 117.